ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/router-internet-exit.md

```diff
@@ -0,0 +1,47 @@
+<!--
+  ~ Copyright (c) 2024 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+=== "Table"
```

Comment:
please add thois table in the documentation in the Routing section: https://github.com/aristanetworks/avd/blob/devel/ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/input-variables.md#routing

Reply:
Done!